scripts/changelog.sh

```diff
@@ -137,22 +150,46 @@ function nextminor {
     rm ./.changes/*.*.*.md
     # Remove all unreleased changes
     rm ./.changes/unreleased/*.yaml
+    # Remove all backported changes
+    rm ./.changes/backported/*.yaml
     # Create a new empty version file for the next minor version
     touch ./.changes/$NEXT_VERSION.md
     
     generate "dev"
 }
 
+# This function is expected to be run on the branch of the last minor release. It will make sure
+# that backports work properly
+function maintenancerelease {
```

Comment:
I'm still learning the release process and what how words are used when talking about it. When looking at the `Is there a Terraform CLI release this week?` doc it looks like maintenance branch is just a different way to refer to any release branch. On the other hand `maintenancerelease` function looks like it'd only be used for backporting from the maintenance/release branch for the previous minor version. Could it be called something like `backportrelease` to make that clearer?

Reply:
>  looks like it'd only be used for backporting from the maintenance/release branch for the previous minor version

The `maintenancerelease` function is used when you create a new maintenance branch to set it up as a target for backporting. Words are hard, I'm not sure if `maintenancerelease` describes it accurately, but I would hope the help text might be a good hint. My second best ideas were `runthisonanewmaintenancebranch` and `runthisonmainafterbranchingoffamaintenancebranch` seemed too verbose 🙈

Reply:
Just so I also understand the meaning - am I right that `maintenancerelease` is for `patch` releases of _previous_ major series? i.e. if we're already working on 1.12 (i.e. `main` == effectively 1.12) like now then we use this to release `1.11.X`, right?

Reply:
Kind of, you would use it when branching of the `v1.11` branch to make sure that branch will handle the backports only